### DISCUSSION TITLE
Hyprland taking more than 50% CPU and everything is glitchy

### DISCUSSION THREAD

**Discussion by violethappyjoy** [hyprwm/Hyprland#7979] [Q&A] [answered]
I updated Nvidia-Drivers and Hyprland, and right after a reboot I saw my CPU temperatures rise and everything is glitchy. When I checked on processes `Hyprland` is using more than 50% of CPU on idle. Also, for some reason it shows 2 workspace on startup even though I am on a single monitor, in which I am not able to use my mouse or key-binds on the first one. I am attaching few screenshots regarding the issue. As, I am new to arch any help will be greatly appreciated.
![image](https://github.com/user-attachments/assets/e88704f3-f45e-4e71-8288-7d53a6fa9ef6)
![image](https://github.com/user-attachments/assets/ca0957a4-4736-466a-a83b-852c9a960ed2)

**Comment by MahouShoujoMivutilde** [comment, marked as answer]
https://github.com/hyprwm/Hyprland/issues/7972#issuecomment-2388846887

**Reply by violethappyjoy** [reply]
Thanks!
